client/lib/performance-tracking/components/performance-tracker-stop.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { stop } from '@automattic/browser-data-collector';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+
+const PerformanceTrackerStop = ( { id } ) => {
+	useEffect( () => {
+		if ( config.isEnabled( 'rum-tracking/logstash' ) ) {
+			stop( id );
+		}
+	} );
```

Comment:
This probably should be:
```js
useEffect( () => {
  ...
}, [] );
```
The empty dependency array ensures that `stop` is called only on mount. We don't want to call it repeatedly on every render, do we?

Reply:
🤦 I always forget about that. Thanks!